assets/scss/application-ie8.scss

```diff
@@ -1,7 +1,7 @@
 $govuk-global-styles: true;
 $path: "/assets/images/";
 
-@import 'govuk/all-ie8';
+@import '../../node_modules/govuk-frontend/dist/govuk/all-ie8';
```

Comment:
This seems like a backward step. Shouldn't we be importing node_modules in to the SASS compiler, so it would be something like `govuk-frontend/dist/govuk/all-ie8`?